app/src/main/java/com/wearerommies/roomie/presentation/ui/mypage/MyRoute.kt

```diff
@@ -149,7 +152,8 @@ fun MyScreen(
 
                     RoomieNavigateButton(
                         type = NavigateButtonType.MY,
-                        text = stringResource(R.string.bookmark_list)
+                        text = stringResource(R.string.bookmark_list),
+                        onClick = { navigateToBookmark() }
```

Comment:
```suggestion
                    onClick = navigateToBookmark
```
람다를 사용하면 별도 객체를 생성하는 것이기에 이렇게 람다없이 호출해주면 메모리 사용량을 줄여줍니다 :-)